shuffle/cli/src/main.rs

```diff
@@ -88,6 +88,15 @@ pub async fn main() -> Result<()> {
     }
 }
 
+#[derive(Debug, StructOpt)]
+struct Command {
+    #[structopt(long, global = true)]
+    home_base_path: Option<PathBuf>,
```

Comment:
home and base seem redundant, can we just call it `home_path` so it looks better in the help documentation. It also matches this LOC better:

```rust
let home = Home::new(normalize_home_path(command.home_base_path).as_path())?;
```